worlds/ladx/Rom.py

```diff
@@ -131,3 +132,47 @@ def get_base_rom_path(file_name: str = "") -> str:
     if not os.path.exists(file_name):
         file_name = Utils.user_path(file_name)
     return file_name
+
+
+def apply_overrides(patch_data: dict) -> None:
+    host_settings = settings.get_settings()
+    option_overrides = host_settings["ladx_options"].get("option_overrides")
+    if not option_overrides:
+        return
+    wrapped_overrides = {
+        "game": LINKS_AWAKENING,
+        LINKS_AWAKENING: option_overrides,
+    }
+    from Generate import roll_settings
+    try:
+        rolled_settings = roll_settings(wrapped_overrides)
+    except:
```

Comment:
A bare except is likely not appropriate here. It should probably at least be `except Exception:` (excluding things like `KeyboardInterrupt`) if there's not a more specific set of exception types that you anticipate here.